ibm_catalog.json

```diff
@@ -530,12 +530,13 @@
               "key": "prefix",
               "type": "string",
               "default_value": "dev",
-              "random_string": {
-                "length": 4
-              },
               "description": "The prefix to add to all resources that this solution creates (e.g `prod`, `test`, `dev`). To skip using a prefix, set this value to null or an empty string. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/prefix.md).",
+              "display_name": "prefix",
```

Comment:
@Aditya-ranjan-16 no need to add a `display_name` here - if not specified it uses the "key" value which is also "prefix"